pkg/manager/internal.go

```diff
@@ -654,11 +646,11 @@ func (cm *controllerManager) Elected() <-chan struct{} {
 	return cm.elected
 }
 
-func (cm *controllerManager) startRunnable(r Runnable) {
+func (cm *controllerManager) startRunnable(ctx context.Context, r Runnable) {
```

Comment:
If I didn't miss anything the `ctx` here is always `cm.internalCtx` so wouldn't it be easier to access that from the `controllerManager` rather than adding a new arg?

Reply:
Yeah good point

Reply:
Done